File: WebApi/InWords.WebApi/InWords.Data.DTO/GameBox/GameInfo.cs
namespace InWords.Data.DTO.GameBox
{
    /// <summary>
    /// Provides information about game Id, creator id, description and title.
    /// Also describe game is that game is available
    /// </summary>
    public class GameInfo
    {
        public int GameId { get; set; }

        public int CreatorId { get; set; }

        public string Description { get; set; }

        public string Title { get; set; }

        public bool IsAvailable { get; set; }
    }
}

File: Android/app/src/main/java/ru/inwords/inwords/game/domain/model/GameInfo.kt
package ru.inwords.inwords.game.domain.model

import android.os.Parcelable
import kotlinx.android.parcel.Parcelize

@Parcelize
data class GameInfo(
    val gameId: Int,
    val description: String,
    val title: String,
    val picture: String,
    val available: Boolean,
    val isCustom: Boolean = false,
    val loading: Boolean = false
) : Parcelable